include/seqan3/std/type_traits

```diff
@@ -87,3 +94,17 @@ using type_identity_t = typename type_identity<t>::type;
 #endif // __cpp_lib_type_identity
 
 } // namespace std
+
+#ifndef SEQAN3_CPP_LIB_COMMON_REFERENCE
+#include <range/v3/utility/common_type.hpp>
+//!cond
+namespace std
+{
+template <typename ...Ts>
+using common_reference = ::ranges::common_reference<Ts...>;
+
+template <typename ...Ts>
+using common_reference_t = ::ranges::common_reference_t<Ts...>;
```

Comment:
CI is complaining that there's no documentation for `\tparam common_reference` and `\tparam common_reference_t`

Reply:
Yeah I used the wrong command to exclude those entities `//\cond` instead of the right syntax `//!\cond`